Make-a-pull-request.md

```diff
@@ -178,8 +178,4 @@ git push origin --delete new-branch-name
 
 ## Step 8: Celebrate
 
-Congratulations, you have contributed to Oppia!
-
-<!--
-TODO(#1): Add link to "How we use pull requests" guide.
--->
+Congratulations, you have contributed to Oppia! To learn more, see our [[guide to PRs at Oppia|Pull-requests-at-Oppia]].
```

Comment:
I don't understand why this is here. Isn't this page the guide in question?

We should link to this guide in the starter instructions, or make it more explicit if it's there already, but I'm not sure why we would surface it after the reader has submitted a PR.

Reply:
There are two pages on PRs: "Make a Pull Request" and "Pull Requests at Oppia." The former is a tutorial on making your first PR, while the second delves into more advanced PR concepts like draft PRs. Since it has more advanced information, I think we should present it as further reading rather than important information that we surface early on

Reply:
Ah ok, sorry, I confused them. That said, I have two suggestions:

- Maybe rename the second wiki page to something a bit more concrete, e.g. "Understanding the Oppia PR Lifecycle", "Common PR workflows at Oppia" or something like that.
- Here, let's be specific about what they're learning more about, so that the reader knows whether to click on the link or not. At the moment the question I have internally is "learn more about what?"